src/utils.js

```diff
@@ -142,3 +142,7 @@ export function generateCollectionPageFileName(title, groupIdentifier) {
   console.log(slugify(title));
   return `${groupIdentifier}-${slugify(title).replace(/[^a-zA-Z-]/g, '')}.md`;
 }
+
+export function generatePermalink(title) {
```

Comment:
`generatePermalink` works fine but the regex for `generatePageFileName` should include numbers as well so it should look like `/[^a-zA-Z0-9-]/g`